### DISCUSSION TITLE
プロフィール画像のアップロードに失敗します

### DISCUSSION THREAD

**Discussion by munaita-0** [increments/qiita-discussions#180] [要望・アイデア・不具合]
![スクリーンショット 2022-03-16 15 39 32](https://user-images.githubusercontent.com/973354/158531512-ed947e90-566f-4ab9-ba21-c53bcacc11fe.png)

ご確認お願いします

**Comment by ohakutsu** [comment]
@munaita-0 
ご報告ありがとうございます！

原因の調査をしてみましたが、提示していただいた画像のように再現ができず原因を特定できていないです...
いくつか画像のアップロードを試したところ、アップロードができており、特定の条件下でのみ発生していると考えています。

差し支えなければ、原因の調査のためアップロードをしようとしていた画像をいただけないでしょうか？
もし、Discussionsで公開したくなければ、
https://support.qiita.com/hc/ja/requests/new
こちらから送っていただけると助かります！:pray:
